utils/S3UtilsMongoClient.js

```diff
@@ -13,37 +13,43 @@ const USERSBUCKET = '__usersbucket';
 const INFOSTORE_TMP = `${INFOSTORE}_tmp`;
 const __COUNT_ITEMS = 'countitems';
 
+const BigIntMax = (...args) => args.reduce((max, current) => {
+    const maxAsBigInt = typeof max === 'bigint' ? max : BigInt(max);
+    const currentAsBigInt = typeof current === 'bigint' ? current : BigInt(current);
+    return maxAsBigInt > currentAsBigInt ? max : current;
```

Comment:
* nit: same as earlier comment: it seems to be this type-check & BigInt conversion should not be needed...
* if in doubt, should probably just call BigInt constructor anyway, to be consistent (if there a perf impact we try to mitigate, then we should do it elsewhere as well...)

```suggestion
    const maxAsBigInt = BigInt(max);
    const currentAsBigInt = BigInt(current);
    return maxAsBigInt > currentAsBigInt ? max : current;
```